include/atama/receiver/node/receiver_node.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#ifndef ATAMA__RECEIVER__NODE__RECEIVER_NODE_HPP_
+#define ATAMA__RECEIVER__NODE__RECEIVER_NODE_HPP_
+
+#include <memory>
+#include <string>
+
+#include "atama/head/head.hpp"
+#include "kansei_interfaces/msg/orientation.hpp"
+#include "ninshiki_interfaces/msg/detected_object.hpp"
+#include "ninshiki_interfaces/msg/detected_objects.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "tachimawari_interfaces/srv/get_joints.hpp"
+
+namespace atama::receiver
+{
+using kansei_interfaces::msg::Orientation;
+using ninshiki_interfaces::msg::DetectedObjects;
+using tachimawari_interfaces::srv::GetJoints;
```

Comment:
i suggest to put it inside the `ReceiverNode`, just to prevent redeclaration of these aliases

Reply:
I forgot it already declare in header

Reply:
doe

Reply:
> done

you haven't put them inside the `ReceiverNode`

Reply:
Do you mean to put it inside cpp file?